Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -859,6 +859,12 @@ class BrowserViewController: UIViewController {
 
         let controller: DismissableNavigationViewController
         controller = DismissableNavigationViewController(rootViewController: libraryViewController)
+        controller.onViewWillDisappear = {
```

Comment:
How come we added this here?

Reply:
This is to prevent a sudden disappearance of the Recently Saved section (while it's visible) after deleting all bookmarks from the bookmarks panel.